### DISCUSSION TITLE
RTL Languages

### DISCUSSION THREAD

**Discussion by vahid07** [templaza/astroid-framework#268] [Q&A]
Hi,

is this framework support RTL style for RTL languages like Persian or Arabic?

**Comment by Chacapamac** [comment]
Same question Here?
@sonvnn

**Reply by vahid07** [reply]
and whats answer?!

**Reply by Chacapamac** [reply]
@sonvnn  are the Astroid Framework have RTL capabilities?

**Reply by sonvnn** [reply]
Sorry for late reply. I just back from Lunar New Year. 
I glad to answer your question. Yes Astroid Framework support RTL languages. You can install Arabic language and test it.

Best Regards,
Sonny

**Reply by Chacapamac** [reply]
Everybody need a brake from time to time. **Happy New Year my Friend!**

This is good news that Astroid is RTL. I never see that on Astroid Specs>

I think this should be put in the Website front Page capabilities “**Astroid has so much more to offer**”
at https://astroidframe.work/

I just Add the “**Documentation**” tag to the discussion
